src/components/02-organisms/Difference/index.js

```diff
@@ -2,12 +2,15 @@ import React, { useState, useCallback, useEffect } from "react";
 import PropTypes from "prop-types";
 import TextField from "@material-ui/core/TextField";
 import Typography from "@material-ui/core/Typography";
+import * as R from "ramda";
```

Comment:
"R.path" only needed.

Reply:
R.path was the cleanest way of safely reaching a nested value from possibly undefined variable. I think it's justified as Ramda is used everywhere anyway.